.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -15,24 +15,35 @@
 # build from the root of this repo:
 FROM gcr.io/cloud-devrel-public-resources/python
 
-ARG SYNTHTOOL_COMMITTISH=a2c9b4a5da2d7f583c8a1869fd2843c206145834
+SHELL [ "/bin/bash", "-c" ]
```

Comment:
Anything requires us to change the default shell to bash?

Reply:
The `nvm` installation script assumes the environment is `bash` based. Using `sh` makes the following line to have no effects.

https://github.com/googleapis/sdk-platform-java/blob/5e71efa12937552faf965cf2520994d2d4e2a74d/.cloudbuild/library_generation/library_generation.Dockerfile#L63